components/Video/PopupVideo/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import ReactPlayer from 'react-player'
+import { PropsWithChildren } from 'react'
+
+interface PopupVideoProps {
+  url: string
+}
+
+const PopupVideo = ({
+  children,
```

Comment:
I'm not an expert on props.children but it doesn't seem to be used anywhere in the code so do you really need it in here?